modules/ROOT/pages/endpoint-https-receive.adoc

```diff
@@ -66,17 +66,6 @@ When you create this API in API Manager:
 NOTE: Using this feature requires an API Manager license.
 |
 
-|HTTPS
-a|Applies to CloudHub deployments.
-
-Indicates whether Partner Manager deploys this endpoint as an HTTPS or HTTP listener:
-
-* If enabled (default), Partner Manager deploys this endpoint as an HTTPS listener.
-* If disabled, Partner Manager deploys this endpoint as an HTTP listener.
-
-When deploying to Anypoint Runtime Fabric, Partner Manager always deploys this endpoint as an HTTP listener. With this type of deployment, you must configure an Ingress controller to provide external consumers with an HTTPS endpoint. For more information, see xref:runtime-fabric::custom-ingress-configuration.adoc[Configuring Ingress for Runtime Fabric].
-|x
-
```

Comment:
Can you confirm that this property is actually removed with this revision as it does not exist in the UI today and also with the release?

Reply:
<img width="1102" alt="Screenshot 2025-04-22 at 1 39 24 PM" src="https://github.com/user-attachments/assets/29efee85-4fa5-4103-a5cf-0d365fffd7fc" />